client/src/components/Fallback/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,177 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`/components/Fallback render correctly 1`] = `
+<Fallback>
```

Comment:
Courtesy of @moshthepitt on article https://kentcdodds.com/blog/effective-snapshot-testing, @HaseemUlHaq lets reduce the size of this snapshot. Succinctly, we should only be interested in testing the props for component `Result` in this case